### ISSUE TITLE
Release pelican-tipue-search v1.0.0 on PyPi

### ISSUE THREAD

**Issue by kdeldycke** [pelican-plugins/tipue-search#2]
Now that the plugin has been migrated out of the monorepo, we can release `pelican-tipue-search` v1.0.0 on PyPi.

TODO:
* [ ] Setup CI/CD by the way of GitHub workflows.
* [ ] Release first revision on PyPi.
* [ ] Flag the plugin as migrated by merging https://github.com/getpelican/pelican-plugins/pull/1296

**Comment by johnmathews** [comment]
I am looking forward to this ❤️

**Comment by dclong** [comment]
Is the code ready and just need to be packed into a Python package?

**Comment by kdeldycke** [comment]
> Is the code ready and just need to be packed into a Python package?

Yes, the code is ready. We need to build a Python package and push it to PyPi. But this requires some checks and setup on GitHub action workflows. We need @justinmayer for that step.

**Comment by thomasgtaylor** [comment]
Any more news on this? I'm looking forward to this plugin as well.

**Comment by justinmayer** [comment]
Hi folks. Regarding the topic of Pelican search functionality, [I posted some bigger-picture thoughts](https://github.com/getpelican/pelican-plugins/pull/1296#issuecomment-787408564) a couple of months ago in another issue thread. Feel free to comment over there with any thoughts you may have.

**Comment by thomasgtaylor** [comment]
Thanks for your update, Justin! It appears that we're waiting on that merge from kdeldycke?

**Comment by justinmayer** [comment]
@t-h-o: No — I posted those thoughts to solicit feedback. Since it's been nearly two months without input on the topic, I will proceed according to the plan I outlined in that comment as soon as I can allocate the time. Shouldn't be very long.

**Comment by thomasgtaylor** [comment]
@justinmayer - My mistake for the misunderstanding. Thank you for all that you do.

**Comment by delip** [comment]
I am here because of this: 
```
$ pip install pelican-tipue-search
ERROR: Could not find a version that satisfies the requirement pelican-tipue-search
ERROR: No matching distribution found for pelican-tipue-search
```

How do I install this plugin?

**Comment by justinmayer** [comment]
@delip: Sorry for the confusion. Since this plugin hasn't actually been published to PyPI yet, you'll need to follow the instructions in the [plugin docs](https://docs.getpelican.com/en/latest/plugins.html#how-to-use-plugins). In short, put the plugin source somewhere in your filesystem and specify its location via the `PLUGINS` setting.

I'll try to rectify this confusing situation as soon as I possibly can. Thanks for your patience.

**Comment by radoeka** [comment]
Would be nice to have this package available via 'pip install pelican-tipue-search'.

**Comment by minchinweb** [comment]
@justinmayer I know you have a lot on your plate, but some movement would be appreciated :)

Right now, the README of the plugin says the best way to install it is off PyPI, but it isn't posted there.

As for the rename branch, it seems [it just needs to be merged](https://github.com/getpelican/pelican-plugins/pull/1296#issuecomment-830802862) (assuming that is still the thought).

If nothing else, could we get the README updated in the meantime?
